### PR TITLE
Update DarkRP wiki links

### PR DESCRIPTION
**Pull request by FlorianLeChat**
The old DarkRP wiki seems will be shutdown in July 31st.

R.I.P old wiki ❤️ 

**EDIT**:  I also updated the Gmod wiki link for [this file](https://github.com/FPtje/darkrpmodification/blob/08f96a8e32afff75fe32ff5d1eef131b85c273c4/lua/darkrp_customthings/ammo.lua#L20).

### CODE REVIEW

**Comment by FPtje** [comment]
Awesome!

File: addon.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "version"           "1"
     "up_date"           "00/00/00"
     "author_name"       "FPtje"
-    "author_email"      "wiki.darkrp.com"
-    "author_url"        "wiki.darkrp.com"
+    "author_email"      ""
+    "author_url"        "https://github.com/FPtje/DarkRP"
```
Comment (Satton2) [inline addon.txt:8]:
```suggestion
    "author_email"      "darkrp.miraheze.org"
    "author_url"        "darkrp.miraheze.org"
```
Aren't these should be like I suggested?

Comment (FlorianLeChat) [reply addon.txt:8]: I did the same thing [here](https://github.com/FPtje/DarkRP/blob/d0f5a6245ea68590dbc712d0780d057010597ae4/darkrp.txt#L9) for information.

Comment (Satton2) [reply addon.txt:8]: Oh... I see. Okay then.

**Pull request merged**